Listening for transport dt_socket at address: 5001
2019-03-03 23:07:34.001: DNA 启动...开始
2019-03-03 23:07:34.003: DNA 启动...java version "1.8.0_111"
2019-03-03 23:07:34.003: DNA 启动...Java(TM) SE Runtime Environment (build 1.8.0_111-b14)
2019-03-03 23:07:34.003: DNA 启动...Java HotSpot(TM) 64-Bit Server VM (build 25.111-b14, mixed mode)
2019-03-03 23:07:34.005: DNA 启动...应用ID：C1E3D50A51F72FED73CBB8109E891B03
2019-03-03 23:07:34.468: DNA 启动...注册JDBC驱动[MySQLDriver5]，驱动版本[5.1]
2019-03-03 23:07:34.596: DNA 启动...注册JDBC驱动[DB2Driver9]，驱动版本[3.1]
2019-03-03 23:07:34.707: DNA 启动...注册JDBC驱动[OracleDriver11]，驱动版本[11.1]
log4j:WARN No appenders could be found for logger (org.eclipse.jetty.util.log).
log4j:WARN Please initialize the log4j system properly.
2019-03-03 23:07:37.347: DNA 启动...实例化信息定义
2019-03-03 23:07:37.387: DNA 启动...装载多语言信息
2019-03-03 23:07:37.391: DNA 启动...准备启动站点
2019-03-03 23:07:37.392: DNA 启动...实例化表定义
2019-03-03 23:07:37.641: DNA 启动...确定表定义对其他元素的引用
2019-03-03 23:07:37.642: DNA 启动...实例化ORM定义
2019-03-03 23:07:37.690: DNA 启动...确定ORM定义对其他元素的引用
2019-03-03 23:07:37.701: DNA 启动...初始化ORM定义
2019-03-03 23:07:37.701: DNA 启动...实例化模型脚本引擎
2019-03-03 23:07:37.724: DNA 启动...初始化群集资源模式
2019-03-03 23:07:37.724: DNA 启动...实例化服务对象
2019-03-03 23:07:37.937: DNA 启动...注册服务调用器
2019-03-03 23:07:38.357: DNA 启动...确定资源定位信息
2019-03-03 23:07:38.361: DNA 启动...确定资源引用信息
2019-03-03 23:07:38.362: DNA 启动...初始化资源
2019-03-03 23:07:38.432: DNA 启动...半启用日志管理器
2019-03-03 23:07:38.432: DNA 启动...初始化服务
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
2019-03-03 23:07:38.541: DNA 启动...启用日志管理器
2019-03-03 23:07:38.541: DNA 启动...站点启动完成
2019-03-03 23:07:38.541: DNA 启动...完毕
osgi> 三月 03, 2019 11:11:27 下午 org.apache.http.impl.execchain.RetryExec execute
信息: I/O exception (java.net.SocketException) caught when processing request to {}->http://ip.taobao.com:80: Connection reset
三月 03, 2019 11:11:27 下午 org.apache.http.impl.execchain.RetryExec execute
信息: Retrying request to {}->http://ip.taobao.com:80
java.util.ConcurrentModificationException
	at java.util.HashMap$HashIterator.nextNode(HashMap.java:1437)
	at java.util.HashMap$KeyIterator.next(HashMap.java:1461)
	at com.squarms.deploy.core.RobotDeployMinaService.testAll(RobotDeployMinaService.java:78)
	at com.squarms.xoms.deploy.core.internal.service.RobotStatusService$RobotTestTaskHandler.handle(RobotStatusService.java:62)
	at com.squarms.xoms.deploy.core.internal.service.RobotStatusService$RobotTestTaskHandler.handle(RobotStatusService.java:1)
	at com.jiuqi.dna.core.resource.ResourceService$SimpleTaskMethodHandler.handle(ResourceService.java:1)
	at com.jiuqi.dna.core.impl.ServiceBase$TaskMethodHandler.handle(ServiceBase.java:1)
	at com.jiuqi.dna.core.impl.ContextImpl.serviceHandleTask(ContextImpl.java:349)
	at com.jiuqi.dna.core.impl.AsyncTaskImpl.workDoing(AsyncTaskImpl.java:146)
	at com.jiuqi.dna.core.spi.work.Work.doWork(Work.java:345)
	at com.jiuqi.dna.core.spi.work.WorkingThread.run(WorkingThread.java:44)
java.util.ConcurrentModificationException
	at java.util.HashMap$HashIterator.nextNode(HashMap.java:1437)
	at java.util.HashMap$KeyIterator.next(HashMap.java:1461)
	at com.squarms.deploy.core.RobotDeployMinaService.testAll(RobotDeployMinaService.java:78)
	at com.squarms.xoms.deploy.core.internal.service.RobotStatusService$RobotTestTaskHandler.handle(RobotStatusService.java:62)
	at com.squarms.xoms.deploy.core.internal.service.RobotStatusService$RobotTestTaskHandler.handle(RobotStatusService.java:1)
	at com.jiuqi.dna.core.resource.ResourceService$SimpleTaskMethodHandler.handle(ResourceService.java:1)
	at com.jiuqi.dna.core.impl.ServiceBase$TaskMethodHandler.handle(ServiceBase.java:1)
	at com.jiuqi.dna.core.impl.ContextImpl.serviceHandleTask(ContextImpl.java:349)
	at com.jiuqi.dna.core.impl.AsyncTaskImpl.workDoing(AsyncTaskImpl.java:146)
	at com.jiuqi.dna.core.spi.work.Work.doWork(Work.java:345)
	at com.jiuqi.dna.core.spi.work.WorkingThread.run(WorkingThread.java:44)
java.util.ConcurrentModificationException
	at java.util.HashMap$HashIterator.nextNode(HashMap.java:1437)
	at java.util.HashMap$KeyIterator.next(HashMap.java:1461)
	at com.squarms.deploy.core.RobotDeployMinaService.testAll(RobotDeployMinaService.java:82)
	at com.squarms.xoms.deploy.core.internal.service.RobotStatusService$RobotTestTaskHandler.handle(RobotStatusService.java:62)
	at com.squarms.xoms.deploy.core.internal.service.RobotStatusService$RobotTestTaskHandler.handle(RobotStatusService.java:1)
	at com.jiuqi.dna.core.resource.ResourceService$SimpleTaskMethodHandler.handle(ResourceService.java:1)
	at com.jiuqi.dna.core.impl.ServiceBase$TaskMethodHandler.handle(ServiceBase.java:1)
	at com.jiuqi.dna.core.impl.ContextImpl.serviceHandleTask(ContextImpl.java:349)
	at com.jiuqi.dna.core.impl.AsyncTaskImpl.workDoing(AsyncTaskImpl.java:146)
	at com.jiuqi.dna.core.spi.work.Work.doWork(Work.java:345)
	at com.jiuqi.dna.core.spi.work.WorkingThread.run(WorkingThread.java:44)
java.util.ConcurrentModificationException
	at java.util.HashMap$HashIterator.nextNode(HashMap.java:1437)
	at java.util.HashMap$KeyIterator.next(HashMap.java:1461)
	at com.squarms.deploy.core.RobotDeployMinaService.testAll(RobotDeployMinaService.java:82)
	at com.squarms.xoms.deploy.core.internal.service.RobotStatusService$RobotTestTaskHandler.handle(RobotStatusService.java:62)
	at com.squarms.xoms.deploy.core.internal.service.RobotStatusService$RobotTestTaskHandler.handle(RobotStatusService.java:1)
	at com.jiuqi.dna.core.resource.ResourceService$SimpleTaskMethodHandler.handle(ResourceService.java:1)
	at com.jiuqi.dna.core.impl.ServiceBase$TaskMethodHandler.handle(ServiceBase.java:1)
	at com.jiuqi.dna.core.impl.ContextImpl.serviceHandleTask(ContextImpl.java:349)
	at com.jiuqi.dna.core.impl.AsyncTaskImpl.workDoing(AsyncTaskImpl.java:146)
	at com.jiuqi.dna.core.spi.work.Work.doWork(Work.java:345)
	at com.jiuqi.dna.core.spi.work.WorkingThread.run(WorkingThread.java:44)
java.util.ConcurrentModificationException
	at java.util.HashMap$HashIterator.nextNode(HashMap.java:1437)
	at java.util.HashMap$KeyIterator.next(HashMap.java:1461)
	at com.squarms.deploy.core.RobotDeployMinaService.testAll(RobotDeployMinaService.java:82)
	at com.squarms.xoms.deploy.core.internal.service.RobotStatusService$RobotTestTaskHandler.handle(RobotStatusService.java:62)
	at com.squarms.xoms.deploy.core.internal.service.RobotStatusService$RobotTestTaskHandler.handle(RobotStatusService.java:1)
	at com.jiuqi.dna.core.resource.ResourceService$SimpleTaskMethodHandler.handle(ResourceService.java:1)
	at com.jiuqi.dna.core.impl.ServiceBase$TaskMethodHandler.handle(ServiceBase.java:1)
	at com.jiuqi.dna.core.impl.ContextImpl.serviceHandleTask(ContextImpl.java:349)
	at com.jiuqi.dna.core.impl.AsyncTaskImpl.workDoing(AsyncTaskImpl.java:146)
	at com.jiuqi.dna.core.spi.work.Work.doWork(Work.java:345)
	at com.jiuqi.dna.core.spi.work.WorkingThread.run(WorkingThread.java:44)
三月 03, 2019 11:27:37 下午 org.apache.http.impl.execchain.RetryExec execute
信息: I/O exception (java.net.SocketException) caught when processing request to {}->http://ip.taobao.com:80: 连接超时 (Read failed)
三月 03, 2019 11:27:37 下午 org.apache.http.impl.execchain.RetryExec execute
信息: Retrying request to {}->http://ip.taobao.com:80
三月 03, 2019 11:28:04 下午 org.apache.http.impl.execchain.RetryExec execute
信息: I/O exception (java.net.SocketException) caught when processing request to {}->http://ip.taobao.com:80: 连接超时 (Read failed)
三月 03, 2019 11:28:04 下午 org.apache.http.impl.execchain.RetryExec execute
信息: Retrying request to {}->http://ip.taobao.com:80
三月 03, 2019 11:28:49 下午 org.apache.http.impl.execchain.RetryExec execute
信息: I/O exception (java.net.SocketException) caught when processing request to {}->http://ip.taobao.com:80: 连接超时 (Read failed)
三月 03, 2019 11:28:49 下午 org.apache.http.impl.execchain.RetryExec execute
信息: Retrying request to {}->http://ip.taobao.com:80
三月 03, 2019 11:28:53 下午 org.apache.http.impl.execchain.RetryExec execute
信息: I/O exception (java.net.SocketException) caught when processing request to {}->http://ip.taobao.com:80: 连接超时 (Read failed)
三月 03, 2019 11:28:53 下午 org.apache.http.impl.execchain.RetryExec execute
信息: Retrying request to {}->http://ip.taobao.com:80
三月 03, 2019 11:28:54 下午 org.apache.http.impl.execchain.RetryExec execute
信息: I/O exception (java.net.SocketException) caught when processing request to {}->http://ip.taobao.com:80: 连接超时 (Read failed)
三月 03, 2019 11:28:54 下午 org.apache.http.impl.execchain.RetryExec execute
信息: Retrying request to {}->http://ip.taobao.com:80
三月 03, 2019 11:29:11 下午 org.apache.http.impl.execchain.RetryExec execute
信息: I/O exception (java.net.SocketException) caught when processing request to {}->http://ip.taobao.com:80: 连接超时 (Read failed)
三月 03, 2019 11:29:11 下午 org.apache.http.impl.execchain.RetryExec execute
信息: Retrying request to {}->http://ip.taobao.com:80
三月 03, 2019 11:29:27 下午 org.apache.http.impl.execchain.RetryExec execute
信息: I/O exception (java.net.SocketException) caught when processing request to {}->http://ip.taobao.com:80: 连接超时 (Read failed)
三月 03, 2019 11:29:27 下午 org.apache.http.impl.execchain.RetryExec execute
信息: Retrying request to {}->http://ip.taobao.com:80
三月 03, 2019 11:29:35 下午 org.apache.http.impl.execchain.RetryExec execute
信息: I/O exception (java.net.SocketException) caught when processing request to {}->http://ip.taobao.com:80: 连接超时 (Read failed)
三月 03, 2019 11:29:35 下午 org.apache.http.impl.execchain.RetryExec execute
信息: Retrying request to {}->http://ip.taobao.com:80
三月 03, 2019 11:29:35 下午 org.apache.http.impl.execchain.RetryExec execute
信息: I/O exception (java.net.SocketException) caught when processing request to {}->http://ip.taobao.com:80: 连接超时 (Read failed)
三月 03, 2019 11:29:35 下午 org.apache.http.impl.execchain.RetryExec execute
信息: Retrying request to {}->http://ip.taobao.com:80
三月 03, 2019 11:29:38 下午 org.apache.http.impl.execchain.RetryExec execute
信息: I/O exception (java.net.SocketException) caught when processing request to {}->http://ip.taobao.com:80: 连接超时 (Read failed)
三月 03, 2019 11:29:38 下午 org.apache.http.impl.execchain.RetryExec execute
信息: Retrying request to {}->http://ip.taobao.com:80
java.util.ConcurrentModificationException
	at java.util.HashMap$HashIterator.nextNode(HashMap.java:1437)
	at java.util.HashMap$KeyIterator.next(HashMap.java:1461)
	at com.squarms.deploy.core.RobotDeployMinaService.testAll(RobotDeployMinaService.java:82)
	at com.squarms.xoms.deploy.core.internal.service.RobotStatusService$RobotTestTaskHandler.handle(RobotStatusService.java:62)
	at com.squarms.xoms.deploy.core.internal.service.RobotStatusService$RobotTestTaskHandler.handle(RobotStatusService.java:1)
	at com.jiuqi.dna.core.resource.ResourceService$SimpleTaskMethodHandler.handle(ResourceService.java:1)
	at com.jiuqi.dna.core.impl.ServiceBase$TaskMethodHandler.handle(ServiceBase.java:1)
	at com.jiuqi.dna.core.impl.ContextImpl.serviceHandleTask(ContextImpl.java:349)
	at com.jiuqi.dna.core.impl.AsyncTaskImpl.workDoing(AsyncTaskImpl.java:146)
	at com.jiuqi.dna.core.spi.work.Work.doWork(Work.java:345)
	at com.jiuqi.dna.core.spi.work.WorkingThread.run(WorkingThread.java:44)
三月 03, 2019 11:30:36 下午 org.apache.http.impl.execchain.RetryExec execute
信息: I/O exception (java.net.SocketException) caught when processing request to {}->http://ip.taobao.com:80: 连接超时 (Read failed)
三月 03, 2019 11:30:36 下午 org.apache.http.impl.execchain.RetryExec execute
信息: Retrying request to {}->http://ip.taobao.com:80
java.lang.RuntimeException: 在开始异步任务[com.squarms.xoms.deploy.core.internal.service.RobotStatusService$RobotTestTask]时发生异常。
	at com.jiuqi.dna.core.impl.AsyncTaskImpl.workBeginning(AsyncTaskImpl.java:82)
	at com.jiuqi.dna.core.spi.work.Work.beforeDoingNoSync(Work.java:279)
	at com.jiuqi.dna.core.spi.work.Work.doWork(Work.java:323)
	at com.jiuqi.dna.core.spi.work.WorkingThread.run(WorkingThread.java:44)
Caused by: com.jiuqi.dna.core.exception.SessionDisposedException: 会话已经销毁或过期
	at com.jiuqi.dna.core.impl.SessionImpl.checkNotDisposedNoSync(SessionImpl.java:641)
	at com.jiuqi.dna.core.impl.SessionImpl.contextCreated(SessionImpl.java:343)
	at com.jiuqi.dna.core.impl.ContextImpl.<init>(ContextImpl.java:1092)
	at com.jiuqi.dna.core.impl.SessionImpl.newContext(SessionImpl.java:505)
	at com.jiuqi.dna.core.impl.AsyncServiceInvoke.workBeginning(AsyncServiceInvoke.java:30)
	at com.jiuqi.dna.core.impl.AsyncTaskImpl.workBeginning(AsyncTaskImpl.java:80)
	... 3 more
